bridge/bridgetest/bridgetest.go

```diff
@@ -119,7 +119,7 @@ func MockFunc(e mockEnvironment) net.Conn {
 		for {
 			d, err := readPbFrame(conB)
 			if err != nil {
-				e.Errorf("Can't read method name")
+				// Conn probably closed
```

Comment:
I was getting an error with go test -race trying to access e from this goroutine. Open to ideas how to fix this better, but it got my tests working